run.sh

```diff
@@ -14,16 +14,17 @@ then
     echo "Great, the simulator is already running!"
 else
     echo "Starting the simulator..."
-    $THIS_DIR/ros/src/styx/unity_simulator_launcher.sh &
+    cd $THIS_DIR/ros/src/styx
+    ./unity_simulator_launcher.sh &
+    cd $THIS_DIR
 fi
 
 # Launch ROS nodes
 docker run --rm=true --tty=true --interactive=true               \
-           --user=$(id --user):$(id --group)                     \
```

Comment:
I really think we should keep this. Otherwise Docker runs as root and can generate artifacts in your host system owned by root. Was the motivation for removing it?

Reply:
*What's

Reply:
just wasn't working for me